a10_octavia/controller/worker/tasks/common.py

```diff
@@ -51,7 +51,7 @@ def readConf(self, section, value):
     def client_factory(self, vthunder):
         axapi_version = acos_client.AXAPI_21 if vthunder.axapi_version == 21 else acos_client.AXAPI_30
         c = acos_client.Client(vthunder.ip_address, axapi_version, vthunder.username, vthunder.password,
-                               timeout=10)
+                               timeout=30)
```

Comment:
Any specific reason, we are increasing timeout ?? is it to avoid ReadTimeoutError ?

Reply:
Yes. These need to be moved a CONF file. And the ability to not timeout should be allowed. On DigitalOcean it's impossible to do these operations within the 10 seconds or so allotted.